pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
+	cmd.Flags().String("cert-file", "", "Provide certificate contents for a secure route. Used by edge and re-encryption terminated routes.")
```

Comment:
Illustration of problem when not using generators to do local creation of objects that are later used in a ```kubectl apply```.

https://github.com/kubernetes/kubernetes/pull/16028#issuecomment-159648935